docs/source/weights_loading.rst

```diff
@@ -108,9 +108,8 @@ But if you don't want to use the values saved in the checkpoint, pass in your ow
 
         def __init__(self, in_dim, out_dim):
             super().__init__()
-            self.in_dim = in_dim
-            self.out_dim = out_dim
```

Comment:
can we not leave it here?
save_hyperparameters will only save in_dim and out_dim, right?
the user should still be able to assign whatever they want to self.

Reply:
yes, he does but we do not want to recommend this practice, right? @williamFalcon

Reply:
people can do whichever but it’s better to use the save call for hparams tracking